core/tests/protractor/editorAndPlayer.js

```diff
@@ -26,9 +26,69 @@ var ExplorationPlayerPage =
 
 describe('Full exploration editor', function() {
   var explorationPlayerPage = null;
+  var parentId2 = null;
+  var refresherExplorationId = null;
+  var currentExplorationId = null;
+  var pathname = null;
+  var parentId1 = null;
 
   beforeAll(function() {
     explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
+    users.createUser('user4@editorAndPlayer.com', 'user4EditorAndPlayer');
+    users.login('user4@editorAndPlayer.com');
+
+    workflow.createAndPublishExploration('Title 1', 'Algebra',
+      'It is a sample exploration.');
+    browser.getCurrentUrl().then(function(url) {
+      pathname = url.split('/');
+      /* In the url a # is added at the end that is not part of exploration
+         ID */
+      parentId1 = pathname[4].slice(0, -1);
+    });
+
+    workflow.createAndPublishExploration('Title 2', 'Algebra',
+      'It is a sample exploration.');
```

Comment:
You have the opportunity to be a bit more descriptive here, in order to help people who are reading the test code. For example, you could write 'This is the original exploration', or 'This exploration is referenced from "Title 1", and leads back there after it's finished.' or something like that.

Reply:
Done